Bundle/WidgetBundle/Controller/WidgetController.php

```diff
@@ -117,6 +117,48 @@ public function newAction($type, $viewReference, $slot = null, $position = null,
     }
 
     /**
+     * @param $widget
+     * @param View $view
+     * @param null $businessEntityId
+     * @param null $namespace
+     * @param null $position
+     * @param null $parentWidgetMap
+     * @param null $slot
+     *
+     * @return string
+     *
+     * @Route("/victoire-dcms/widget/entity/create/{viewReference}/{businessEntityId}/{type}/{slot}/{parentWidgetMap}/{position}", name="victoire_core_widget_entity_create", defaults={"widget_id":null, "slot":null, "businessEntityId":null, "position":null, "parentWidgetMap":null, "_format": "json"})
+     * @Route("/victoire-dcms/widget/entity/update/{viewReference}/{businessEntityId}/{type}/{widget_id}/{slot}/{parentWidgetMap}", name="victoire_core_widget_entity_edit", defaults={"widget_id":null, "slot":null, "businessEntityId":null, "position":null, "parentWidgetMap":null, "_format": "json"})
+     * @ParamConverter("widget", class="VictoireWidgetBundle:Widget", options={"id" = "widget_id"})
+     */
+    public function entityAction($viewReference, $slot, $position, $parentWidgetMap, $businessEntityId, $type, Widget $widget = null)
```

Comment:
@Charlie-Lucas I don't understand this name, please try to find a better and more explicit name